src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js

```diff
@@ -83,7 +72,6 @@ describe('AuthorizationForm ', () => {
     // the same amount of tag is kept because it was not added
     expect(cloudTags.querySelectorAll('li').length).toBe(emails.length + 1);
     expect(errors).toBeInTheDocument();
-
     // success when add second tag (simulated with enter event)
     await fireEvent.change(input, { target: { value: tagToAdd2 } });
```

Comment:
In this part, you should use userEvent.clear(input) before userEvent.type method. This for clean input field before typing. 
![image](https://user-images.githubusercontent.com/84402180/131347575-62d47045-c683-47b4-bd8d-f1640d849950.png)